src/components/Dashboard/MyRatings.js

```diff
@@ -0,0 +1,187 @@
+    let ratingsData = [];
+    $.ajax({
+      url: url,
+      jsonpCallback: 'pxcd',
+      dataType: 'jsonp',
+      jsonp: 'callback',
+      crossDomain: true,
+      success: function(data) {
+        if (data.rated_skills) {
+          for (let i of data.rated_skills) {
+            let skill_name = Object.keys(i)[0];
+            ratingsData.push({
+              skill_name: skill_name,
+              skill_star: i[skill_name].stars,
+              rating_timestamp: i[skill_name].timestamp,
+            });
+          }
+          self.setState({
+            ratingsData,
+          });
+        }
+        self.setState({
+          loading: false,
+        });
+      },
+      error: function(err) {
+        self.setState({
+          loading: false,
+          openSnackbar: true,
+          msgSnackbar: "Error. Couldn't rating data.",
+        });
+      },
+    });
+  };
+
+  render() {
+    let ratingsData = this.state.ratingsData;
+    return (
+      <div>
+        {this.state.loading ? (
+          <div className="center">
+            <CircularProgress size={62} color="#4285f5" />
+            <h4>Loading</h4>
+          </div>
+        ) : (
+          <div className="table-wrap">
+            <Table className="table-root" selectable={false}>
+              <TableHeader displaySelectAll={false} adjustForCheckbox={false}>
+                <TableRow>
+                  <TableHeaderColumn>Skill Name</TableHeaderColumn>
+                  <TableHeaderColumn>Rating</TableHeaderColumn>
+                  <TableHeaderColumn>Timestamp</TableHeaderColumn>
+                </TableRow>
+              </TableHeader>
+              <TableBody displayRowCheckbox={false}>
+                {ratingsData.map((skill, index) => {
+                  return (
+                    <TableRow key={index}>
+                      <TableRowColumn style={{ fontSize: '16px' }}>
+                        {(
+                          skill.skill_name.charAt(0).toUpperCase() +
+                          skill.skill_name.slice(1)
+                        ).replace(/[_-]/g, ' ')}
+                      </TableRowColumn>
+                      <TableRowColumn style={{ fontSize: '16px' }}>
+                        {skill.skill_star}
+                      </TableRowColumn>
+                      <TableRowColumn>
+                        {this.parseDate(skill.rating_timestamp)}
+                      </TableRowColumn>
+                    </TableRow>
+                  );
+                })}
+                <TableRow />
+              </TableBody>
+            </Table>
+          </div>
+        )}
+        {ratingsData.length === 0 &&
+          !this.state.loading && (
+            <div>
+              <div className="center">
+                <br />
+                <h2>
+                  You have not rated any skill, go to{' '}
+                  <a
+                    href="https://github.com/fossasia/susi_skill_cms/blob/master/docs/Skill_Tutorial.md"
```

Comment:
I'm not sure if this should redirect to the Skill Tutorial file. We can't rate any skill there. How is that related here? Let's wait for more opinions anyways. Rest LGTM :+1:

Reply:
Ah sorry, I took that from MySkills and forgot to replace the link, that should redirect to index page.

Reply:
Fixed